maven-plugin/src/main/java/org/sonatype/nexus/maven/staging/StagingDeployMojo.java

```diff
@@ -106,6 +109,12 @@ private void doExecute() throws MojoFailureException, MojoExecutionException {
     catch (MojoExecutionException e) {
       throw e;
     }
+    catch (dv ex) {
+        Optional<String> message = ex.b();
```

Comment:
I have no idea why this needs to be obfuscated but an error message like this is very poor:

```
[ERROR] Failed to execute goal org.sonatype.plugins:nxrm3-maven-plugin:1.0.3:staging-deploy (default-cli) on project XYZ: Upload component was unsuccessful (400 response from server): Bad Request -> [Help 1]
```

Only executing Maven with `--debug` unveiled the real error:
```
[DEBUG] http-outgoing-3 << "[ {[\n]"
[DEBUG] http-outgoing-3 << "  "id" : "*",[\n]"
[DEBUG] http-outgoing-3 << "  "message" : "The assets 3 and 4 have identical coordinates"[\n]"
[DEBUG] http-outgoing-3 << "} ]"
```